recipes/mlflow/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "mlflow" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c2c76d072f4735a229333095506acb3d6ec0305656ee7fc468e9b60a0e8d7b8a
+
+build:
+  number: 0
+  entry_points:
+    - mlflow=mlflow.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [not win]
```

`mlflow` doesn't work on windows due to the missing set of dependencies. Thus this condition must be the other way around.

```suggestion
  skip: True  # [win]
```